docs/source/api/standalone.mdx

```diff
@@ -0,0 +1,212 @@
+---
+title: "API Reference: startStandaloneServer"
+api_reference: true
+---
+
+This API reference documents the `startStandaloneServer` function.
+
+## Overview
+
+This `startStandaloneServer` function helps you get started with Apollo Server quickly. This function is recommended for all projects that don't require serverless support or a particular Node.js framework (such as Fastify). Under the hood, the `startStandaloneServer` function uses Apollo Server 4's Express integration (i.e., [`expressMiddleware`](./express-middleware)). 
+
+Because it sets helpful defaults, this function is less configurable than other Apollo Server integrations. Complex projects might eventually need to [swap to using `expressMiddleware`](#swapping-to-expressmiddleware) (this process is straightforward).
+
+## `startStandaloneServer`
+
+The `startStandaloneServer` function accepts two arguments, the first **required** argument is the instance of `ApolloServer` to begin listening for incoming requests:
+
+<MultiCodeBlock>
+
+```ts
+import { ApolloServer } from '@apollo/server';
+import { startStandaloneServer } from '@apollo/server/standalone';
+
+const server = new ApolloServer({ typeDefs, resolvers });
+
+// `startStandaloneServer` returns a `Promise` with the
+// the URL that the server is listening on.
+const { url } = await startStandaloneServer(server); //highlight-line
+``` 
+
+</MultiCodeBlock>
+
+The `startStandaloneServer` function's second optional argument is an object for configuring your server's options, which can contain the following properties:
+
+### Options
+
+<table class="field-table">
+  <thead>
+    <tr>
+      <th>Name /<br/>Type</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+
+<tbody>
+<tr>
+
+<td>
+
+  ##### `context`
+
+  `Function`
+
+</td>
+
+<td>
+
+An optional asynchronous [`context` initialization function](../data/resolvers#the-context-argument). 
+
+The `context` function should return an object, which gets passed to every resolver that executes for a particular operation. This enables resolvers to share helpful context values, such as a database connection.
+
+The `context` function receives `req` and `res` options which are `http.IncomingMessage` and `http.ServerResponse` types.
+
+</td>
+
+</tr>
+
+<tr>
+
+<td>
+
+  ##### `listen`
+
+  `Object`
+
+</td>
+
+<td>
+
+  An optional `listen` configuration option. The `listen` option accepts an object with the same properties as the [`net.Server.listen` options object](https://nodejs.org/api/net.html#serverlistenoptions-callback).
```

Comment:
Maybe we should be even more explicit that it only takes the options object, not any argument to that function (ie you can't pass `listen: 1234`)

(well technically that will work on a JS level, but not a TS level, and passing multiple arguments that way won't work)

Reply:
We explicitly say, "with the same properties as the `net.Server.listen` options object", so it feels pretty clear to me. I can add italics for emphasis if you think that's clearer:

> An optional `listen` configuration object. The `listen` option accepts an object with the same properties as the [`net.Server.listen` _options object_](https://nodejs.org/api/net.html#serverlistenoptions-callback).